core/src/datayoga_core/blocks/relational/read/block.py

```diff
@@ -13,7 +13,7 @@
 
 class Block(DyProducer):
 
-    def init(self, context: Context):
+    def init(self, context: Optional[Context] = None):
```

Comment:
but here it is really not optional... I guess we can add a check and raise an exception if it is None